whylog/config/parsers.py

```diff
@@ -67,9 +67,12 @@ def from_dao(cls, serialized_parser):
 class RegexParserFactory(object):
     @classmethod
     def create_from_intent(cls, parser_intent):
+        convertions = dict(
+            (group_id, group.converter) for (group_id, group) in parser_intent.groups.items()
+        )
```

Comment:
As I know, we don't keep in Config group contents. Config wanted to have line_content. group_content is just a substring of line_content.